lang/strings_english.txt

```diff
@@ -1187,7 +1187,7 @@ $s_issue_id = 'Issue #';
 $s_recently_visited = 'Recently Visited';
 $s_priority_abbreviation = 'P';
 $s_note_user_id_label = 'Note By:';
-$s_filter_match_type = 'Match Type';
+$s_filter_match_type_label = 'Match Type:';
```

Comment:
I'd rather have consistency on this kind of things (ie label strings being identified as such), as it reduces the maintenance overhead in the long run. 

We could easily rename the strings for all languages but I'd like to check with @siebrand first to avoid any issues with translatewiki updates. As an alternative we could do this in 2 steps, first introduce new string, wait for translations to be updated then remove the old one.